Sources/CocoaLumberjack/include/DDDispatchQueueLogFormatter.h

```diff
@@ -27,6 +27,7 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  *  Log formatter mode
  */
+__attribute((deprecated))
```

Comment:
There is a nice (modern) macro `API_DEPRECATED` to declare deprecations, which also takes a message. 😉

Reply:
I don't think we can use that here: `API_DEPRECATED` needs platform version parameter which is of no use here.
We can use `__attribute((deprecated))` with message too but I'm wondering which message to use.

Reply:
Ah, you're right. That's what I get for only doing a quick repo search instead of looking at the source code. GitHub found a usage of API_DEPRECATED where the platforms were on the next line, so I thought it could be used without platforms. 🤦‍♂️

A message would still be nice, but I see you already added those. Thanks!